src/com/android/launcher3/ExtendedEditText.java

```diff
@@ -141,4 +142,18 @@ public void reset() {
         }
         hideKeyboard();
     }
+
+    /**
+     * This method allows to force AutoFill availability.
+     *
+     * @param forceDisableAutoFill whether to disable AutoFill or not.
+     */
+    public void setAutoFill(boolean forceDisableAutoFill) {
```

Comment:
I know I have no right to judge the code. But in the coding aspect, it's really misleading. The `setAutoFill` should accept the autofill enable value, not the disable value. In the `getAutofillType` method just switch the ternary operator. Then the comment can be removed where it's used _(CustomBottomSheet#108)_ as well.